_includes/ssi-ml/schedule.html

```diff
@@ -1,18 +1,14 @@
 <h3>Parallel sessions.</h3>
-There will be parallel workshops on day 2, 3, & 4.
-FIXME: Add instructions for how to choose sessions.
-
-FIXME: Timing assumes 9:00 - 13:00
-
-FIXME: Add other stuff during the kickoff week?
+These are the parallel hands-on workshops on Tuesday, Wednesday, & Thursday of the SSI-ML kickoff week.
 
+Each morning we start with a plenary introduction in zoom before we move into the parallel sessions on Discord.
```

Comment:
This is unfortunately not true. We will start in the channel right away, because switching to zoom for 15 min is not practical. @mkuzak can you confirm?

Reply:
OK, I will just leave it out. This will hopefully sort itself out today.